src/utils/archive.ts

```diff
@@ -199,7 +200,59 @@ export async function createPublicDrive(
     txs: [driveTx, rootFolderTx]
   };
 }
+export async function sendArDriveFee(
+  keyfile: JWKInterface,
+  arPrice: number,
+  arweave: Arweave
+) {
+  try {
+    // Get the latest ArDrive Community Fee from the Community Smart Contract
+    let fee = arPrice * ((await getArDriveFee()) / 100);
+
+    // If the fee is too small, we assign a minimum
+    if (fee < 0.00001) {
```

Comment:
Consider the following:

1. Factor out the default minimum tip as a constant.
2. Fetching the real minimum tip setting from the ArDrive PST contract
3. Using Math.max(getArDriveMinimumTip(), fee) rather than an if block

Reply:
[Updated](https://github.com/th8ta/ArConnect/pull/33/commits/f76faa99ec9bfb48f340cbe9b4857e805e4e2582)